examples/planets/pyproject.toml

```diff
@@ -2,16 +2,3 @@
 package_name = "planets"
 project_name = "planets"
 project_version = "0.18.2"
-
-[tool.isort]
```

Comment:
Why have you removed that?

Reply:
We already have a pre-commit and `pyproject.toml` in the global source. I don't think this makes any sense. In addition, I want to make the sample project as small as possible.